### DISCUSSION TITLE
Disable zeek log

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#2221] [Unsupported Versions] [answered]
I'm trying to prevent certain logs, like x509 for instance, from being ingested and shown in kibana.  The docs say visit https://github.com/Security-Onion-Solutions/securityonion/blob/master/pillar/zeeklogs.sls and use that example to move from enabled to disabled.  The link doesn't work, and I can't find a file with an example.

**Comment by dougburks** [comment, marked as answer]
Try `sudo so-zeek-logs`. If that works for you, let us know and we'll get the docs updated.

**Comment by gcsrellis** [comment]
Worked great, thanks for the help!

**Reply by dougburks** [reply]
Added https://docs.securityonion.net/en/2.3/so-zeek-logs.html#so-zeek-logs and referenced it at https://docs.securityonion.net/en/2.3/filebeat.html.
